Sources/_CryptoExtras/RSA/RSA.swift

```diff
@@ -98,6 +98,14 @@ extension _RSA.Signing {
             }
         }
 
+        /// Construct an RSA public key with the specified parameters.
+        ///
+        /// Only the BoringSSL backend provides APIs to create the key from its parameters so we first create a BoringSSL
+        /// key, serialize it to PEM format, and then construct a platform specific key from the PEM representation.
+        public init<D: ContiguousBytes>(n: D, e: D) throws {
```

Comment:
All of these need multiple generic type parameters. There's no particularly good reason to force each parameter to be the same type.

Reply:
Yep. I was thinking the same thing. Originally I had these take hex strings but I think bytes is probably better. Will make this more flexible 👍

Reply:
Updated so that each generic parameter can be a different concrete type conforming to `ContiguousBytes` and added a test that would have failed before, but now passes.